tembo-cli/src/cmd/apply.rs

```diff
@@ -838,6 +856,41 @@ fn get_postgres_config(
     postgres_config
 }
 
+async fn get_loadable_libraries(
+    mut shared_preload_libraries: Vec<Library>,
+    maybe_extensions: Option<&Vec<ControllerExtension>>,
+) -> Result<Vec<Library>, anyhow::Error> {
+    if let Some(extensions) = maybe_extensions {
+        let trunk_projects_url =
+            "https://registry.pgtrunk.io/api/v1/trunk-projects?extension-name=";
+
+        for ext in extensions.iter() {
+            let response = reqwest::get(format!("{}{}", trunk_projects_url, ext.name))
+                .await?
+                .text()
+                .await?;
+
+            let trunk_projects: Vec<TrunkProject> = serde_json::from_str(&response).unwrap();
```

Comment:
also I think this unwrap looks unsafe, if the response from the API does not fit the TrunkProject struct for some reason